src/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -222,8 +222,23 @@ class UpdateConfig(CommandBase):
 
     def run(self, args, unknown_args):
         unknown_args += ('-e', '{"_should_update_formplayer_in_place": true}')
-        return commcare_cloud(args.env_name, 'ansible-playbook', 'deploy_localsettings.yml',
+        rc = commcare_cloud(args.env_name, 'ansible-playbook', 'deploy_localsettings.yml',
                               tags='localsettings', branch=args.branch, *unknown_args)
+        if rc == 0 and ask("Would you like to run Django checks to validate the settings?"):
```

Comment:
how long do these take? would it be worthwhile to just always run the checks?

Reply:
`check_services` is the longest - that can take up to 30s.